dashboard/lib/projects_list.rb

```diff
@@ -109,8 +112,18 @@ def get_published_project_data(project)
         # Note that we are using the new :project_type field rather than extracting
         # it from :value. :project_type might not be present in unpublished projects.
         type: project[:project_type],
-        publishedAt: project[:published_at]
+        publishedAt: project[:published_at],
+        studentName: student_name(project),
+        studentAge: student_age(project),
       }.with_indifferent_access
     end
+
+    def student_age(project)
+      ((Date.today - project[:birthday]) / 365).to_i
+    end
+
+    def student_name(project)
+      project[:name] ? project[:name][0] : ''
```

Comment:
Our [`User#initial`](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/user.rb#L797-L800) method upcases. Should we do the same here?

The method name is misleading, it seems more appropriately named `student_initial`.

Should we with `student_age` and `student_name` be reusing the `User` methods?